public/css/themes/github/LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
The files are covered by the LICENSE file already present in the directory they were moved to. These files are part of that project.